typo3-ter/cal/2010-03-02.yaml

```diff
@@ -0,0 +1,8 @@
+title:     "TYPO3-SA-2010-005: Blind SQL Injection vulnerability in extension Calendar Base (cal)"
+link:      http://typo3.org/teams/security/security-bulletins/typo3-extensions/typo3-sa-2010-005/
+cve:
```

Comment:
Imo adding `~` would make clear that the CVE link was left empty on purpose and was not simply forgotten. So I agree with @stof on this.